LibGAP.jl/src/julia_to_gap.jl

```diff
@@ -103,16 +99,14 @@ function julia_to_gap(obj::Dict{T,S}, recursive::Val{Recursive}=Val(false), recu
     if haskey(recursion_dict,obj)
         return recursion_dict[obj]
     end
-    nr_entries = obj.count
-    keys = Array{T,1}(undef,nr_entries)
-    entries = Array{S,1}(undef,nr_entries)
-    i = 1
-    for (x,y) in obj
-        keys[i] = x
-        entries[i] = y
-        i += 1
-    end
-    record = GAP.Globals.CreateRecFromKeyValuePairList(julia_to_gap(keys,Val(true)),julia_to_gap(entries, recursive, recursion_dict))
+
+    k = collect(T, keys(obj))
+    v = collect(S, values(obj))
+
+    k = julia_to_gap(k, Val(true))
```

Comment:
NITPICK: No need to put the `Val(true)` here. The keys have to be `Symbol` or `String`, and both conversions ignore the recursion anyway.

Reply:
To be honest, I don't understand this line at all, but I copied verbatim form what you wrote... I'd have expected `k = julia_to_gap(k, recursive, recursion_dict)`

Reply:
In a sense, this makes sense. But the list of keys `k` (which I am not sure is the name I used) is a list of strings anyway. You can do recursion here, but it does not matter at all.